skits/feature_extraction.py

```diff
@@ -69,17 +69,30 @@ def transform(self, X, y=None):
         Xt2 = self.ar2.transform(X)
         return Xt1 - Xt2
 
+class MovingAverageTransformer(BaseEstimator, TransformerMixin):
```

Comment:
The [moving average](https://en.wikipedia.org/wiki/Moving-average_model) term of ARIMA is the difference between the time series value and the model's prediction for that value. This requires solving differential equations which is incompatible with using scikit-learn estimators. 

Maybe change the name of this transformer to `RollingMeanTransformer` so that it's not confused with the ARIMA definition (and it'll also follow pandas' terminology).

Reply:
Good point! Guess I should go reread Hyndman's book :). Will do.